masterlist.yaml

```
@@ -15906,16 +15924,18 @@ plugins:
     clean:
       - crc: 0xB4C1E773
         util: 'SSEEdit v4.0.3'
-  - name: 'ClockworkTitleRemover.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+  - name: 'ClockworkSuperiorSorting.esp'
+    req: [ *PapyrusExtender, *SKSE ]
```

To keep formatting consistent across masterlists.

```suggestion
    req:
      - *PapyrusExtender
      - *SKSE
```

---

I got that formatting from another plugin (it only had PapyrusExtender if I remember correctly, so it might be different for just one mod), so that might be worth reviewing as well.

---

If it's just one item, it can (& we prefer it to) be in-lined (e.g., `req: [ *PapyrusExtender ]`).